git-hooks/commit/errors.js

```diff
@@ -0,0 +1,87 @@
+const TYPES = require('./types')
```

Comment:
Please see comment above concerning `semicolons`. 

I recommend adding the semicolons.